docker/venice-client/sample-data/multi-dc-configs/batch-push-job.properties

```diff
@@ -0,0 +1,15 @@
+venice.urls=http://venice-controller.dc-parent.venicedb.io:5555
+venice.discover.urls=http://venice-controller.dc-parent.venicedb.io:5555
+venice.store.name=test-store
+input.path=/opt/venice/sample-data/batch-push-data
+key.field=key
```

Comment:
FYI, this is now the default, so no need to specify it anymore. Same for value.

Reply:
Ack.